compliance/instance_types/list_new_instance_types.pt

```diff
@@ -0,0 +1,395 @@
+  run_script $js_filter_new_instance_types, $ds_aws_instance_types, $ds_github_aws_instance_types, $ds_azure_instance_types, $ds_github_azure_instance_types, $ds_google_machines_list, $ds_google_SQL_tier_list, $ds_github_google_instance_types
+end
+
+###############################################################################
+# Script
+###############################################################################
+
+script "js_filter_one_subscription", type: "javascript" do
+  parameters "subscriptions"
+  result "filtered_subscriptions"
+  code <<-EOS
+    var subscriptionsList = [];
+    if(subscriptions.length > 0){
+      subscriptionsList.push({
+        "subscriptionId": subscriptions[0].subscriptionId
+      })
+    }
+    filtered_subscriptions = subscriptionsList;
+  EOS
+end
+
+script "js_filter_one_project", type: "javascript" do
+  parameters "projects"
+  result "filtered_projects"
+  code <<-EOS
+    var projectsList = [];
+    if(projects.length > 0){
+      projectsList.push({
+        "projectId": projects[0].projectId
+      })
+    }
+    filtered_projects = projectsList;
+  EOS
+end
+
+script "js_filter_new_instance_types", type:"javascript" do
+  parameters "aws_instance_types", "github_aws_instance_types", "azure_instance_types", "github_azure_instance_types", "google_instance_type", "google_SQL_tyres_list", "github_google_instance_types"
+  result "instance_types"
+  code <<-EOS
+    var new_instance_types_found = [];
+    
+    // Finding new instance types in AWS
+    _.each(aws_instance_types, function(value, key){
+      var aws_instance_type_check_in_github = github_aws_instance_types[value.instanceType]
+      if(aws_instance_type_check_in_github == null){
+        new_instance_types_found.push({
+          "instanceType": value.instanceType,
+          "cloudProvider": "AWS"
+        })
+      }
+    })
+
+    // Finding new instance types in Azure
+    var  azure_unique_instance_types =     _.uniq(_.map(azure_instance_types, function(d){return d.instanceType}));
+    _.each(azure_unique_instance_types, function(value, key){
+      var azure_instance_type_check_in_github = github_azure_instance_types[value]
+      if(azure_instance_type_check_in_github == null){
+        new_instance_types_found.push({
+          "instanceType": value,
+          "cloudProvider": "Azure"
+        })
+      }
+    })
+
+    // Finding new instance types for google
+    //combining the vm types and db tiers
+    var machines_list_obj={}
+    var github_list_machines = []
+    _.each(google_instance_type, function(machine){
+      if(machines_list_obj[machine.name]==null){
+	      machines_list_obj[machine.name]=machine;
+	    }
+    })
+
+    _.each(google_SQL_tyres_list, function(sql){
+      if(machines_list_obj[sql.name]==null){
+	      //filtering out 1st gen db tiers
+	      if(sql.name.indexOf("D")==0){
+        }
+		    else{
+		      machines_list_obj[sql.name]=sql;
+		    }
+	    }
+	  })
```

Comment:
Check the line indentation.

Reply:
The format is correct in the pt file but in Github, it is not showing the correct format.

Reply:
See if it contains tab space.

Reply:
Done